src/Wizard/WizardSteps/WizardStep/WizardStep.tsx

```diff
@@ -75,6 +78,9 @@ function WizardStepLabel({
   // errorText overrides helperText
   if (errorText) {
     stepLabelProps.optional = errorText;
+  }
+
+  if (error || errorText) {
```

Comment:
What was the background behind introducing error? If we're now relying on that to determine if we've got an error then shouldn't we only be checking for that rather than that or errorText? Presumably if error is false and errorText is set to some value then I shouldn't see the error?

Reply:
I introduced this because I was forced to set text to indicate the error but in the UI I'm displaying the error in the field/field group. I maintained the ability to use the error text to also set the error  as not to force a breaking change to how its currently being used. 

If we are making a breaking change I would suggest an error boolean and helperText to match how we display errors in inputs as we're replacing the helperText with errorText anyway. I ran a quick check and only BUILD is currently using the errorText in VIRTO

Reply:
I have updated as per discussion